pommerman/graphics.py

```diff
@@ -396,13 +403,20 @@ def board_right(self, x_offset):
         return constants.BORDER_SIZE + (
             self._board_size * self._tile_size) + x_offset
 
+    def get_bomb_life(self, row, col):
```

Comment:
Currently, this is returning None if there is no bomb and that (row, col). Is that a possibility? If so, does the resource manager handle it ok?

Reply:
This is only called if the item is a bomb (see lines 296-300). I didn't have a clear way of doing this. I could remove it from the class and have it so you pass in the bombs array. something like a helper function.

Reply:
I'm going to approve this because it's a low pri concern, but just a heads up that it's not the best practice as it's not very safe. This might cause an error in the future and we should remain aware of that.